### DISCUSSION TITLE
Multiple platform streaming

### DISCUSSION THREAD

**Discussion by s4v0h34d** [owncast/owncast#1613] [General Owncast discussion]
Hi, sorry but I couldn't find an answer to this question, if somebody already answered, please link me to the discussion and sorry.
My question is:
is there a way to stream from OBS to my owncast server AND to Twitch (for example)?
I would love to migrate from a commercial community to my own broadcasting platform, but I'd love to do it gradually, so for a little while I would try and stream my content both on Twitch and on my owncast server, in order to make my follower migrate and get used to this new and awesome world.
Thanks again for everything you're doing and keep up the great work
s

**Comment by jeyemwey** [comment]
Hi, you can try out https://github.com/muesli/prism which acts as a simple splitter. There have been multiple users who use prism to serve Twitch and Owncast at the same time.

Good luck with that and happy holidays!

**Reply by s4v0h34d** [reply]
thank you so much! I will try it out!
